### ISSUE TITLE
[cloud-tensorflow-mnist]:

### ISSUE THREAD

**Issue by ran5515** [googlecodelabs/feedback#143]
#Step 6
the last code
test_data={X: mnist.test.images, Y_: mnist.test.labels}
a,c = sess.run([accuracy, cross_entropy], feed=test_data)

should be 
a,c = sess.run([accuracy, cross_entropy], feed_dict=test_data)
